server/model/monitor.js

```diff
@@ -45,6 +45,12 @@ class Monitor extends BeanModel {
         if (showTags) {
             obj.tags = await this.getTags();
         }
+
+        if (includeStatus) {
+            const heartbeat = await Monitor.getPreviousHeartbeat(this.id);
+            obj.status = heartbeat.status === 1 ? "up" : "down";
```

Comment:
Would be nice to support the third state which is maintenance as well

Reply:
@mathiash98 This PR was created where there were only two states. I will rebase and ass the third state.